util/tlgen/README.md

```diff
@@ -1,4 +1,4 @@
 # TL-UL Crossbar RTL generator
 
 `tlgen.py` is a generation script for building TileLink-based RTL crossbar files.
-See the full [reference manual](../doc/rm/crossbar_tool.md) for details on this utility script.
+See the full [reference manual](../../doc/rm/crossbar_tool/index.md) for details on this utility script.
```

Comment:
probably want a relref here too

({{<relref "doc/rm/crossbar_tool">}})

Reply:
It's a README.md which we've been typically allowing to be vanilla Markdown instead of hugo-ified.